ansible/request-ocp-roks-play/examples/roks-vars.yml

```diff
@@ -7,8 +7,7 @@ dataCenter: wdc04 # Provide the data center (zone) to deploy to
 defaultPoolSize: 2 # Set the size of the default worker pool
 entitlement: cloud_pak # Set entitlement to "cloud_pak" when deploying a cloud pak to the cluster otherwise you will be charged for OCP licenses
 hardware: shared # shared for virtual workers
-icaccount: 1234567890123456 # Target the account to use
-kubeVersion: 4.4_openshift # kube version to use 
+kubeVersion: 4.6_openshift # kube version to use 
```

Comment:
should kubeversion be part of defaults in the request role

Reply:
IBM Cloud will default (today to 4.6.x) if that variable is not provided but I have it in the role as a default so the role doesn't break when run with an empty value. I believe it will fail without it. If there's a better way to handle that I can.